src/api/public/apidocs-new/components/parameters/request_withfullhistory.yaml

```diff
@@ -0,0 +1,9 @@
+in: query
+name: withfullhistory
+schema:
+  type: string
+description: |
+  Add the request and review history.
```

Comment:
What about "Include" or "Display" instead of "Add"?

Reply:
Fine. Changed to "Include".